package.json

```diff
@@ -17,9 +17,9 @@
     "yeoman-generator"
   ],
   "dependencies": {
-    "chalk": "^3.0.0",
-    "lodash": "^4.17.15",
-    "yeoman-generator": "4.5.0",
+    "chalk": "^4.1.0",
```

Comment:
I guess this is a breaking change.

Reply:
Yes, nothing to worry about. This is a dev tool, the templates still are Node 8 compatible.